### PR TITLE
feat: Automatically remove maintainers from members to resolve conflicts

### PR DESCRIPTION
**Pull request by mariux**
- chore!: drop inputs from outputs and add docs
- chore!: drop support for broken terraform versions
- feat!: Add support for module_enabled
- feat: Automatically remove maintainers from members to resolve conflicst on the fly
- chore: prepare 0.8.0 release

### CODE REVIEW

**Review by kassianh — DISMISSED** [review]
lgtm

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.8.0]
```
Comment (soerenmartius) [inline CHANGELOG.md:10]: please add scope, 

e.g.

```
## Breaking Change

## Added 
- ...

```

**Review by soerenmartius — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (mariux) [inline CHANGELOG.md:10]:
```suggestion
## [0.8.0]

### Added

```

Comment (mariux) [reply CHANGELOG.md:10]: good catch... done.

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+
+### Added
+
```
Comment (soerenmartius) [inline CHANGELOG.md:13]:
```suggestion
```

**Review by soerenmartius — APPROVED** [review]
lgtm

**Pull request merged**